<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageUtils</class>
 <widget class="QMainWindow" name="ImageUtils">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageUtils</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="LabelXCoordinate">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="LabelYCoordinate">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string>透明度</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QSlider" name="horizontalSlider">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QPushButton" name="ButtonTop">
             <property name="text">
              <string>top</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QPushButton" name="ButtonLeft">
               <property name="text">
                <string>left</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ButtonRight">
               <property name="text">
                <string>right</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonBottom">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="text">
              <string>bottom</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="TestGraphicsView" name="TemplateView"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Rectify</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Rate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="RectifyRateInput">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Rotate</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Angle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="RotateAngleInput">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Scale</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QPushButton" name="ScaleModeButton">
                    <property name="text">
                     <string>Increment Mode</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>X-Axis</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="XScaleInput">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Y-Axis</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="YScaleInput">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="ImageGenerateButton">
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ResetParaButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCaliLite">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionRead_Template_Image"/>
    <addaction name="actionRead_Source_Image"/>
    <addaction name="actionSave_Template_Image"/>
   </widget>
   <addaction name="menuCaliLite"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRead_Template_Image">
   <property name="text">
    <string>Read Template Image</string>
   </property>
  </action>
  <action name="actionRead_Source_Image">
   <property name="text">
    <string>Read Source Image</string>
   </property>
  </action>
  <action name="actionSave_Template_Image">
   <property name="text">
    <string>Save Template Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TestGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>TestGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
